.gitignore

```diff
@@ -1,5 +1,6 @@
-/target/
+/**/.classpath
+/**/.project
```

Comment:
This is not a good idea since it will clear the .project file with each branch.
And this file does not hurt anyway.